scripts/ampHtmlValidator/index.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-console */
+const fetch = require('node-fetch');
+const amphtmlValidator = require('amphtml-validator');
+const { getPageUrls } = require('../../cypress/support/helpers/getPageUrls');
+
+const environment = 'local';
+const isSmoke = true;
+const baseUrl = 'http://localhost:7080';
+const pageTypes = [
+  'frontPage',
+  'articles',
+  'liveRadio',
+  'photoGalleryPage',
+  'mostReadPage',
+  'liveRadio',
```

Comment:
Can we add mediaAssetPage / onDemandRadio to this list? Or is that a separate PR?

Reply:
Will do